### ISSUE TITLE
Process suspended after a few seconds

### ISSUE THREAD

**Issue by ZooD3** [eltoncezar/Spofy#23]
After launching from the .exe, the process appears for a few seconds before being suspended. No window is launched.
![image](https://user-images.githubusercontent.com/39098385/43473594-855fcc78-94e8-11e8-8c70-cd3094a3a989.png)

**Comment by eltoncezar** [comment]
Yeah, for me too. I'll take a look, looks like something changed on the Spotify side...

**Comment by eltoncezar** [comment]
As you can see in the README, I use SpotifyAPI-NET for all Spotify related operations. But Spotify changed the way they handle stuff... So, until we have more information, or an workaround, I think this project is broken :(
The guys at SpotifyAPI-NET are aware of that, you can see their discussion here: https://github.com/JohnnyCrazy/SpotifyAPI-NET/issues/254

**Comment by ZooD3** [comment]
Oh bummer, I was hoping I'd finally found a working Spotify mini player!
Thanks anyway, good luck!

**Comment by eltoncezar** [comment]
And you found it! At least it was working as last week... Sorry about that man, I will keep an eye in this issue, and notify here if we found a solution.

**Comment by eltoncezar** [comment]
I will keep that issue open, so others can find it easily.

**Comment by eltoncezar** [comment]
I'm looking into it. Maybe I'll do a complete rewrite using the new SpotifyAPI-NET.
